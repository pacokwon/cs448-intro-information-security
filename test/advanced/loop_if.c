int main()
{
    int x = 10;
    int y = 100;


    if(x / y == 2)
    {
        int z = x * 100;
    }
    else if(x + y >= 10)
    {
        int z = y / x;
        int k = 0;

        for(int i = 0; i < x; i++)
        {
            z = (x + x) / i;
            k += z;
        }

        int interval_safe = x / k;
    }
    else
    {
        int k = x - y;

        int err = (x + y) / k;
    }

}
