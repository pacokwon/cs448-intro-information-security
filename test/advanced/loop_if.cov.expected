7,10
7,14
7,8
11,15
11,19
11,13
13,19
16,13
16,26
16,9
18,20
